sys-power/apcupsd/metadata.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<maintainer type="person" proxied="yes">
```

Comment:
I wouldn't drop him entirely.. he commented in https://bugs.gentoo.org/937496 and you merged a PR from him in https://github.com/gentoo/gentoo/pull/38681, right?

Reply:
Yeah, that's right. But AFAICS, that the only one commit otherwise? Anyway, I let him there and add myself only as co. That' fine for me.